README.md

```diff
@@ -502,6 +502,18 @@ task_definition:
         string: string
       labels:
         string: string
+  efs_volumes:
+     - name: string                     // Required
```

Comment:
since this is a top-level ecs-params.yml example, including `REQUIRED` doesn't seem right (since the parent obj is itself optional). Suggest only commenting on fields that have finite valid values (example: line 498)